docs/en/drivers/ios-uiautomation.md

```diff
@@ -0,0 +1,225 @@
+## The UIAutomation Driver for iOS
+
+> **Note**: This driver is _DEPRECATED_ and should not be used unless
+> absolutely necessary. The information in this doc may not keep up to date
+> with reality, and the driver will be removed in a future version of Appium.
+> To begin iOS automation with Appium today, please use the [XCUITest
+> Driver](/docs/en/drivers/ios-xcuitest.md) instead.
+
+Appium's former method for iOS app automation was based on `UIAutomation`, an
+Apple-provided framework that shipped with the iOS SDK until iOS 10, when it
+was removed. `UIAutomation` was one of the tools included in Apple's
+`instruments` profiling system, and provided a JavaScript API that ran
+synchronously in the context of a single app. The Appium UIAutomation driver
+established an asynchronous, session-based WebDriver front end for this API.
+
+Development of the UIAutomation driver is done at the
+[appium-ios-driver](https://github.com/appium/appium-ios-driver) repo.
+
+### Requirements and Support
+
+(In addition to Appium's [general requirements](#TODO))
+
+* Xcode 7 or lower.
+* iOS simulators or devices with version 9.3 or lower.
+* All versions of Appium ship with this driver.
+* For correct functioning of the driver, see additional setup below.
+
+### Usage
+
+The way to start a session using the UIAutomation driver is to set the
+`platformName` [capability](#TODO) in your [new session request](#TODO) to the
+value of `iOS`. Of course, you must also include appropriate `platformVersion`,
+`deviceName`, and `app` capabilities, at a minimum.
+
+### Capabilities
+
+The UIAutomation driver supports a number of standard [Appium
+capabilities](/docs/en/writing-running-appium/caps.md), but has an additional
+set of capabilities that work for this driver only (see the "iOS" section of
+the aforementioned doc).
+
+To automate Safari instead of your own application, leave the `app` capability
+empty and instead set the `browserName` capability to `Safari`.
+
+### Commands
+
+To see the various commands Appium supports, and specifically for information
+on how the commands map to behaviors for the UIAutomation driver, see the [API
+Reference](#TODO).
+
+### Simulator Setup
+
```

Comment:
Perhaps a short bit about the fact that only a single Simulator can be running, and automated, at a time?